source/index.ts

```diff
@@ -123,9 +125,19 @@ function getTypedContextTypes(schemas: QuerySchemasResponse[]) {
   const TypedContextSubtype = `export type TypedContextSubtype = keyof TypedContextSubtypeMap;`;
   return { TypedContextSubtypeMap, TypedContextSubtype };
 }
-// Call the generate function with command line arguments if this module is being run directly
-if (import.meta.url === `file://${process.argv[1]}`) {
+
+const isRunningDirectlyViaCLI = !require.main;
```

Comment:
`isRunningDirectlyViaCLI` would be true if it was imported into another ESM node script and ran from there.  I'm a bit confused why the current implementation does not work for you.  Which shell and version of Node are you using?

Could you console.log `import.meta.url` and `process.argv[1]`?

Reply:
Although, we're started removing the option to import it into other node scripts anyways since we couldn't really find a proper use case for that, already removed it from the readme. So we could just remove the check completely.

Reply:
I think it is because the path is a symlink (see repro steps) due to NPM link. I might have been linking it too early when trying to find the cause of a network error (the one I made another PR for over at the API repo).

So maybe it doesn't mean anything.

But good idea with removing the check entirely. I'll do that! 😍

Reply:
I was on Node 18 by the way.

Also could only test the positive case (that it was not working), but couldn't test the case if running as import as mentioned in the PR. Didn't know how (most likely if the documentation for it was then removed, makes total sense!) 😅

Reply:
I can totally see how symlinks could mess with the current implementation. Wasn't something that had been given much thought :)

But yeah, just removing that check would solve all problems. If anyone surfaces a need to include it in another script further down the line then it can be revisited then.

Reply:
And just FYI, `import.meta.url` gave `"file:///Users/username/Documents/src/ftrack-ts-schema-generator/dist/ftrack-ts-schema-generator.js"`, and `process.argv[1]` gave `"/Users/username/Documents/src/project-name/src/lambda/node_modules/.bin/ftrack-ts-schema-generator"`.

`"/Users/username/Documents/src/project-name/src/lambda"` was where my own project was located.

The other was where I had cloned the generator.

Reply:
Makes total sense - I removed it now! 👍